docs/gateway-configuration/network-hardware.md

```diff
@@ -0,0 +1,26 @@
+# Hardware Tab
+
+Each interface in the Network page contains inside its specific box the tab `Hardware`: this collects the most significant information about the current state of the interface.
+
+In this way the user can have a quick and complete view of what are the settings and physical specifications of a network interface.
+
+![](images/hardware-tab-image.png)
+
+These information are respectively:
+
+- **Status**: represents the current state of the network interface, from Kura’s point of view. Some examples of status are `UNMANAGED`, `ACTIVATED`, `FAILED`, `UNKNOWN`.
```

Comment:
"the current state of the network interface, from the framework point of view. Some examples of status are `UNMANAGED`, `ACTIVATED`, `FAILED`, `UNKNOWN`.